src/docs/implementation/SwateManual/Docs03-Building-Blocks.md

```diff
@@ -0,0 +1,522 @@
+        | 10,00   day                 	| day  	| UO                                 	| http://purl.obolibrary.org/obo/UO_0000033 	|
+
+        </p>
+        </details>
+
+    - Done! 🎉
+
+<br>
+
+# Building Block Types
+
+Swate and the ARC feature a set of six defined Building Block types. They can be split into input, descriptive and output Building Blocks and you can find additional information on the [nfdi4plant](https://nfdi4plants.github.io/AnnotationPrinciples/) website.
+
+## Input Building Block/Source Name
+
+**Definition:** This is the only input column type and each *Annotation Table* **MUST** contain this column once. 
+
+**Expected Values:** Unique identifier for input.
+- **MUST** be unique in the overlaying ARC.
+- **MAY** be sample name (think what you would write on the sample in the lab) or file name.
+- If the same sample/extract/file is referenced anywhere else in the ARC it **MUST** always be referenced by this identifier.
+
+**Examples:**
+
+<details><summary>Example | Sample pooling.</summary>
+<p>
+
+| Source Name  	| Parameter [...]  	| Sample Name  	|
+|---	|---	|---	|
+| MinimalChlamy1  	| ...  	|  MinimalChlamyHarvested1 	|
+| MinimalChlamy2  	| ...  	|  MinimalChlamyHarvested1 	|
+| MinimalChlamy3  	| ...  	|  MinimalChlamyHarvested2 	|
+| MinimalChlamy4  	| ...  	|  MinimalChlamyHarvested2 	|
+
+- 👀 This example shows a sample pooling, in which two source samples are pooled into one output sample. This can also be done in reverse, where one source can be split into multiple output samples.
+
+</p>
+</details>
+
+<details><summary>Example | From physical sample to data file.</summary>
+<p>
+
+| Source Name  	| Parameter [...]  	| Data File Name  	|
+|---	|---	|---	|
+| MinimalChlamyProteins1  	| ...  	|  minimal1.mzLite 	|
+| MinimalChlamyProteins2  	| ...  	|  minimal2.mzLite 	|
+| MinimalChlamyProteins3  	| ...  	|  minimal3.mzLite 	|
+| MinimalChlamyProteins4  	| ...  	|  minimal4.mzLite 	|
+
+</p>
+</details>
+
+<details><summary>Example | From data file to data file.</summary>
+<p>
+
+| Source Name  	| Parameter [...]  	| Data File Name  	|
+|---	|---	|---	|
+|  minimal1.mzLite  	| ...  	|  minimal1.prot 	|
+|  minimal2.mzLite  	| ...  	|  minimal2.prot 	|
+|  minimal3.mzLite  	| ...  	|  minimal3.prot 	|
+|  minimal4.mzLite  	| ...  	|  minimal4.prot 	|
+
+</p>
+</details>
+
+<br>
+
+## Descriptive Building Blocks
+
+**General Definition:** 
+- These Building Blocks **MAY** be added multiple times, but each Building Block combination **MUST** be unique.
+- These Building Blocks **MUST** concist of either <u>three</u> columns, or <u>four</u> columns in case they are created with the unit option. 
+    - Of which one column, the <u>main column</u> is visible, while the othere <u>reference columns</u> are hidden by default. 
+<details><summary>Additional in-depth information.</summary>
```

Comment:
if i look at this now it might be good to add a sub details logic so table headers and table body can be closed and opened like `Additional in-depth information`. But this is highly optional